telluric/collections.py

```diff
@@ -32,6 +34,40 @@
 CONCURRENCY_TIMEOUT = int(os.environ.get('TELLURIC_LIB_CONCURRENCY_TIMEOUT', 600))
 
 
+def dissolve(collection, aggfunc=None):
+    # type: (BaseCollection, Optional[Callable[[list], Any]]) -> GeoFeature
+    """Dissolves features contained in a FeatureCollection and applies an aggregation
+    function to its properties.
+
+    """
+    if aggfunc:
+        new_properties = dict.fromkeys(collection.attribute_names)
+        for name in new_properties:
+            # noinspection PyBroadException
+            try:
+                new_properties[name] = aggfunc(collection.get_values(name))
+
+            except Exception:
+                # We just do not use these results
+                pass
+
+    else:
+        new_properties = {}
+
+    if 'raster_url' in collection.attribute_names:
+        final_raster = merge_all([feature.get_raster() for feature in collection])
+        dest_file = tempfile.NamedTemporaryFile(suffix=".tiff")
+        final_raster.save(dest_file)
```

Comment:
be careful, 
if a raster is defined with a remote url what do you expect to be done here? to fetch the entire raster from the INTERNET?